src/OSPSuite.Core/Domain/Services/ObserverBuilderTask.cs

```diff
@@ -52,7 +57,7 @@ public void CreateObservers(IBuildConfiguration buildConfiguration, IModel model
          }
          finally
          {
-            _allContainers.Clear();
+              _allContainerDescriptors = null;
```

Comment:
That should be enough to remove the reference here. We could clear but I don't think it's necessary